src/sentry_scope.c

```diff
@@ -74,6 +74,7 @@ get_scope(void)
     g_scope.contexts = sentry_value_new_object();
     sentry_value_set_by_key(g_scope.contexts, "os", sentry__get_os_context());
     g_scope.breadcrumbs = sentry_value_new_list();
+    sentry_value_append(g_scope.breadcrumbs, sentry_value_new_int32(1));
```

Comment:
I think it would be much better to move this as a special case (`l->len == 0`) into `sentry__value_append_ringbuffer()`, so that we do not spread the invariants over multiple locations.

Reply:
Makes sense, I adapted the append code & the conversion to list (since we can now have a ringbuffer without items if the first value is never added) 
https://github.com/getsentry/sentry-native/pull/1060/commits/190a4a5d1dc195d5b038634f83902f79c8e84b0b#diff-c47c40069b4dfa1e5cc367dcd84f2c8845913ad9dff9ff82686139981f95c5abR858-R860